packages/wdio-junit-reporter/src/index.js

```diff
@@ -40,14 +38,15 @@ class JunitReporter extends WDIOReporter {
                 continue
             }
 
-            const specFileName = runner.specs[index]
+            // The specs array will always return only one spec file since each reporter is invoked per spec file
```

Comment:
yes but in spec file can be two describes, no? It was introduced in https://github.com/webdriverio/webdriverio/pull/3818

Reply:
Hmm I'm not really sure what problem #3818 is solving. I don't see any way the same single runner instance can be invoked by multiple spec files.

Reply:
spec file is one, but it has N describes like
```

describe("first", ()=> {
    before(() => {
        browser.url("http://google.com");
    });

    it('should do', () => {

    });
});

describe("second", ()=> {
    before(() => {
        browser.url("http://google.com");
    });

    it('should noo do', () => {

    });
});
```

Reply:
I'm still not really sure what the problem is/was. It's doing the looping of the describes right above in `this.suites` so they can still have however may they want. The example above is working for me.